### PR TITLE
Fixes for Windows ARM64

### PR DESCRIPTION
**Pull request by sean-mcmanus**
Still testing this...

### CODE REVIEW

File: Extension/README.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ The extension has platform-specific binary dependencies, therefore installation
 `cpptools-linux-aarch64.vsix` | Linux ARM 64-bit
 `cpptools-osx.vsix` | macOS
 `cpptools-win32.vsix` | Windows 64-bit & 32-bit
+`cpptools-win-arm64.vsix` | Windows ARM64
```
Comment (michelleangela) [inline Extension/README.md:59]: Also update the other README.md in the root folder

Comment (sean-mcmanus) [reply Extension/README.md:59]: Good catch.

**Review by Colengms — APPROVED** [review]

**Comment by sean-mcmanus** [comment]
The TypeScript changes seem okay so far, but the ARM64 binary isn't running and I haven't tested on 32-bit Windows yet.

**Comment by sean-mcmanus** [comment]
I'm checking this in, but tag parsing isn't working yet...

File: Extension/src/githubAPI.ts [hunk 2]
```diff
@@ -114,7 +114,15 @@ function getArrayOfBuilds(input: any): Build[] {
 export function vsixNameForPlatform(info: PlatformInformation): string {
     const vsixName: string | undefined = function(platformInfo): string | undefined {
         switch (platformInfo.platform) {
-            case 'win32': return 'cpptools-win32.vsix';
+            case 'win32':
+                switch (platformInfo.architecture) {
+                    case 'x86_64': return 'cpptool-win32.vsix'; // TODO: Change to cpptools-win64?
```
Comment (bobbrow) [inline Extension/src/githubAPI.ts:119]: It looks like this is misspelled

Comment (bobbrow) [reply Extension/src/githubAPI.ts:119]: This needs to be fixed or else insiders will be broken for x64.

Comment (sean-mcmanus) [reply Extension/src/githubAPI.ts:119]: yikes, good catch.

**Pull request merged**